.all-contributorsrc

```diff
@@ -31,6 +31,16 @@
         "doc"
       ]
     },
+    {
```

Comment:
no need to add this part, I will do it independently via the allcontributors bot :)

Reply:
Ahh, good to know!